.gitignore

```diff
@@ -1,4 +1,5 @@
 # Default ignored files
 /shelf/
 /workspace.xml
-.idea
\ No newline at end of file
+.idea
+fashi_on_the_way.db
```

Comment:
Nice :)
Pay attention that you committed the venv directory and its not something you want to do(it contains thousands of files):
<img width="343" alt="Screen Shot 2021-07-07 at 0 11 46" src="https://user-images.githubusercontent.com/45973026/124667334-efe3d780-deb7-11eb-97a3-b4c3843d957a.png">

To remove the folder from this branch (without delete it from your local project) stash your local changes, open cmd, go to your project path:
```cd <your project path>```
and run these commands:
``` 
git checkout databae_fix // go to this branch
git rm -r venv //remove the venv folder from git
git commit . -m "Remove unnecessary directory" //commit your changes
git push origin database_fix //push the local changes to github  ```

Now, add the venv directory to .gitignore